nbbuild/antsrc/org/netbeans/nbbuild/CheckModuleConfigs.java

```diff
@@ -80,8 +80,8 @@ public void setNbroot(File f) {
             }
             allClusters.put(m.group(1), splitToList((String) clusterDef.getValue(), clusterDef.getKey()));
         }
-        allClusters.get("experimental").removeAll(allClusters.get("betauc")); // intentionally a superset
-        allClusters.get("betauc").removeAll(allClusters.get("stableuc")); // ditto
+//        allClusters.get("experimental").removeAll(allClusters.get("betauc")); // intentionally a superset
+//        allClusters.get("betauc").removeAll(allClusters.get("stableuc")); // ditto
```

Comment:
I would remove this completely. The whole setup around `experimental` and `betauc` looks like a hack and it is IMHO better to remove it and if needed in the future we might have better ideas.